laravel-app/app/Http/Controllers/PostController.php

```diff
@@ -62,8 +62,10 @@ public function store(UserPostRequest $request): RedirectResponse
     /*
     * Display specific post.
     */
-    public function show(UserPost $post): View
+    public function show(User $user, UserPost $post): View
```

Comment:
You added $user in the argument but you also uses auth()->user() ehy??

Reply:
nice, catch sir

Reply:
Haha partida sa cp ako nagrereview nyan

Reply:
Removed the unused argument on post.show, sir. Thank you ❤️